README.md

```diff
@@ -578,11 +618,65 @@ Forked from [AirBNB's style guide](https://github.com/airbnb/javascript)
   var subtract = Function('a', 'b', 'return a - b');
   ```
 
+  - [7.11](#7.11) <a name="7.11"></a> Spacing in a function signature.
+
+    > Why? Consistency is good, and you shouldn’t have to add or remove a space when adding or removing a name.
+
+    ```javascript
+    // bad
+    const f = function(){};
+    const g = function (){};
+    const h = function() {};
+
+    // good
+    const x = function () {};
+    const y = function a() {};
+    ```
+
+  - [7.12](#7.12) <a name="7.12"></a> Never mutate parameters. eslint: [`no-param-reassign`](http://eslint.org/docs/rules/no-param-reassign.html)
+
+    > Why? Manipulating objects passed in as parameters can cause unwanted variable side effects in the original caller.
+
+    ```javascript
+    // bad
+    function f1(obj) {
+      obj.key = 1;
+    };
+
+    // good
+    function f2(obj) {
+      const key = Object.prototype.hasOwnProperty.call(obj, 'key') ? obj.key : 1;
```

Comment:
these two examples are different. One is a setter, the other is a reader. To mirror, you would want `f2` to be something like

```js
function f2(obj) {
  return merge(obj, { key: 1 })
}